lib/devtools/config.rb

```diff
@@ -169,8 +171,8 @@ class Mutant < self
 
     # Devtools configuration
     class Devtools < self
-      FILE                      = 'devtools.yml'.freeze
-      DEFAULT_UNIT_TEST_TIMEOUT = 0.1  # 100ms
+      FILE                      = 'devtools.yml'
+      DEFAULT_UNIT_TEST_TIMEOUT = 0.1 # 100ms
```

Comment:
Unlike rubocop (in the new version) wants it, the two whitespaces are intentional for post comments in my code. Please change rubocop settings here to agree to this.

Reply:
ok, will fix it today